src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,243 @@
+    cbsd_fad_records_sas_test_harness_1_iteration_0 = generateCbsdRecords([sas_test_harness_device_4,sas_test_harness_device_5],
+                                                                          [[grant_request_2, grant_request_3], [grant_request_4]])
+
+    # Generate Cbsd FAD Records for SAS Test Harness 0, iteration 1
+    cbsd_fad_records_sas_test_harness_0_iteration_1 = generateCbsdRecords([sas_test_harness_device_2,sas_test_harness_device_3],
+                                                                          [[grant_request_1],[grant_request_2]])
+
+    # Generate Cbsd FAD Records for SAS Test Harness 1, iteration 1
+    cbsd_fad_records_sas_test_harness_1_iteration_1 = generateCbsdRecords([sas_test_harness_device_6], [[grant_request_5, grant_request_6]])
+   
+    # SAS Test Harnesses configuration  
+    sas_test_harness_0_config = {
+        'sasTestHarnessName': 'SAS-TH-1',
+        'hostName': 'localhost',
+        'port': 9001,
+        'serverCert': "certs/server.cert",
+        'serverKey': "certs/server.key",
+        'caCert': "certs/ca.cert"
+    }
+    sas_test_harness_1_config = {
+        'sasTestHarnessName': 'SAS-TH-2',
+        'hostName': 'localhost',
+        'port': 9002,
+        'serverCert': "certs/server.cert",
+        'serverKey': "certs/server.key",
+        'caCert': "certs/ca.cert"
+    }
+
+    # Generate SAS Test Harnesses dump records for multiple iterations   
+    sas_test_harness_0_dump_records_iteration_0 = {
+        'cbsdRecords': cbsd_fad_records_sas_test_harness_0_iteration_0
+    }
+    sas_test_harness_0_dump_records_iteration_1 = {
+        'cbsdRecords': cbsd_fad_records_sas_test_harness_0_iteration_1
+    }
+    sas_test_harness_1_dump_records_iteration_0 = {
+        'cbsdRecords': cbsd_fad_records_sas_test_harness_1_iteration_0
+    }
+    sas_test_harness_1_dump_records_iteration_1 = {
+        'cbsdRecords': cbsd_fad_records_sas_test_harness_1_iteration_1
+    }
+
+    # Create the actual config.
+    iteration1_config = {
+        'cbsdRequests': [cbsd_records_iter_1_domain_proxy_0,cbsd_records_iter_1_domain_proxy_1],
+        'protectedEntities': protected_entities_iter_1,
+        'dpaActivationList': [dpa_1, dpa_2],
+        'dpaDeactivationList': [],
+        'sasTestHarnessesData': [sas_test_harness_0_dump_records_iteration_0,sas_test_harness_1_dump_records_iteration_0]
+    }
+    iteration2_config = {
+        'cbsdRequests': [cbsd_records_iter_2_domain_proxy_0,cbsd_records_iter_2_domain_proxy_1],
+        'protectedEntities': protected_entities_iter_2,
+        'dpaActivationList': [dpa_3],
+        'dpaDeactivationList': [dpa_1],
+        'sasTestHarnessesData': [sas_test_harness_0_dump_records_iteration_1,sas_test_harness_1_dump_records_iteration_1]
+    }
+    config = {
+        'iterationData': [iteration1_config, iteration2_config],
+        'sasTestHarnesses': [sas_test_harness_0_config, sas_test_harness_1_config],
```

Comment:
`sasTestHarnessConfigs` to match with FAD.2.

Reply:
Modified   to "sasTestHarnessConfigs"